### ISSUE TITLE
Project: parsing GTFS real-time feeds and visualising

### ISSUE THREAD

**Issue by tslumley** [AU-BURGr/UnConf2017#9]
I see from https://gtfsrt.api.translink.com.au/ that Queensland has real-time GTFS feeds for public transport. 

I've been using the Auckland feed for simple visualisations (@tuureiti on Twitter, and https://www.stat.auckland.ac.nz/~tlum005/buses/) and for a more complex project on predicting arrival times at stops.  However, the Auckland feed is in JSON, which is easy to parse. The Queensland feed uses the much more efficient protobuf format, so we'd need to make R bindings for, say, the C++ protobuf library.

**Comment by jonocarroll** [comment]
I think someone from last year was looking into doing this (Daniel Thomas/@Daniel-t?).

Anyway, here's someone's attempt from another hackathon: https://www.youtube.com/watch?v=W2XQOI9bzGY

**Comment by tslumley** [comment]
Yes, there's a lot of examples like that hackathon one of 'where are the buses now'. 
I'm more looking to be able to do data analysis, though.

**Comment by Daniel-t** [comment]
I did work this out eventually, for a govhack event that wasn't able to attend. Here's the vis that I pulled together using the data for a day (at 10 minute intervals. http://i.imgur.com/BLfzEAu.jpg

The biggest issue I have is that the translink feed isn't compatible the the current gtfs specification. I had to mess with things to decode it to Json /xml
